apps/taiga-lumbermill/src/dashboards/iot/components/contacts/contacts.component.less

```diff
@@ -0,0 +1,14 @@
+tui-tabs {
+    display: flex;
+    justify-content: space-around;
+}
+
+[tuiCardLarge] [tuiCell][data-size='l'] {
```

Comment:
try to not override card/cell properties, 
remove this, add gap: var(--tui-gap) for list, make bottom-border using `box-shadow`